com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -98,16 +98,42 @@ public class PostProcessPass : ScriptableRenderPass
 
         Material m_BlitMaterial;
 
-        public PostProcessPass(RenderPassEvent evt, PostProcessData data, Material blitMaterial)
+        public PostProcessPass(RenderPassEvent evt, PostProcessData data, ref PostProcessParams postProcessParams) : this(evt, data)
+        {
+            m_BlitMaterial = postProcessParams.blitMaterial;
+
+            // Texture format pre-lookup
+            const FormatUsage usage = FormatUsage.Linear | FormatUsage.Render;
+            if (SystemInfo.IsFormatSupported(postProcessParams.requestHDRFormat, usage))
+            {
+                // TODO: we could get render target format from
+                // static `UniversalRenderPipeline.asset.hdrFormat´ but that would means a static tight dependency to the pipeline asset.
+
```

Comment:
Feedback: Do we want to rely asset being always available? and add dependencies? I'd say no.

Reply:
Data is the resources.